src/main/java/org/com/itpple/spot/server/common/validator/EnumValidator.java

```diff
@@ -16,15 +16,18 @@ public void initialize(EnumValid constraintAnnotation) {
     public boolean isValid(String value, ConstraintValidatorContext context) {
         boolean result = false;
         Object[] enumValues = this.annotation.enumClass().getEnumConstants();
-        if (enumValues != null) {
-            for (Object enumValue : enumValues) {
-                if (value.equals(String.valueOf(enumValue))) {
-                    result = true;
-                    break;
-                }
 
+        if (enumValues == null) {
+            return false;
+        }
+
+        for (Object enumValue : enumValues) {
+            if (value.equals(String.valueOf(enumValue))) {
+                result = true;
+                break;
             }
         }
+
         return result;
```

Comment:
```suggestion
        return Arrays.stream(enumValues)
            .anyMatch(enumValue -> value.equals(String.valueOf(enumValue)));
```
이렇게 수정하면 변수 하나를 줄일 수 있지 않을까요?